lib/propolis/src/intr_pins.rs

```diff
@@ -27,6 +27,8 @@ pub trait IntrPin: Send + Sync + 'static {
             self.deassert();
         }
     }
+
+    fn import_state(&self, is_asserted: bool);
```

Comment:
I know the rest of this trait is missing doc comments, but for something like this which is less obvious in its use, a good explanation would be useful for both consumers and implementers.

Reply:
How do you feel about the comment from 1412f9cddb1c52ef8d0ecc610e1def32cb0e08a7?  I'd love any suggestions on how to describe this more clearly.

Reply:
I suppose I would focus more on the edge not being visible to the _consumer_.  In the case of `FuncPin`, we're not emitting guest-sourced events to the greater hypervisor (propolis-server).  Including a specific example of the kernel-vmm-managed state being left unchanged as well would probably also be appropriate.

> This method is intended for use when importing a guest.

I'd probably move this up in the description, since it's key info about the motivations here.

Reply:
Okay, I gave it another pass. Hopefully this one isn't too wordy.